sql/linestring.go

```diff
@@ -150,13 +150,13 @@ func (t LineStringType) SQL(dest []byte, v interface{}) (sqltypes.Value, error)
 		return sqltypes.NULL, nil
 	}
 
-	pv, err := t.Convert(v)
+	v, err := t.Convert(v)
 	if err != nil {
 		return sqltypes.Value{}, nil
 	}
 
-	//TODO: pretty sure this is wrong, pv is not a string type
-	val := appendAndSliceString(dest, pv.(string))
+	buf := SerializeLineString(v.(LineString))
+	val := appendAndSliceString(dest, HexForString(string(buf)))
```

Comment:
You should try using `hex.EncodeToString([]byte)` from the standard library and avoid having to write our own HexForString function.  The only difference is, I believe, that `hex.EncodeToString` doesn't add the '0x' prefix automatically.